.travis.yml

```diff
@@ -1,25 +0,0 @@
-language: ruby
-
-before_install:
-  - gem i bundler
-
-bundler_args: --without benchmark --jobs=3 --retry=3
-
-matrix:
-  include:
-    - rvm: 2.4.9
-    - rvm: 2.5.7
-    - rvm: 2.6.5
-    - rvm: 2.7.0
-    - rvm: jruby-9.2.9.0
-      env: JRUBY_OPTS=--debug
-      jdk: openjdk8
-    - rvm: ruby-head
-    - rvm: jruby-head
-      env: JRUBY_OPTS=--debug
-      jdk: openjdk8
```

Comment:
Similar question, can one have jdk switching on github actions/do you know? We had some trouble where we had to pin it to older JDKs due to problems with new JDKs and I'd dislike not having control over that - or well at least I'd be concerned.

Reply:
That's a good question I don't know the answer too, but I'll research.

Reply:
I fixed this by manually installing the requested jdk.